From 47e32d1ecd7a3d57503b281be3c3705e10fa403f Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Thu, 5 Jan 2023 06:42:19 +0000
Subject: [PATCH] socs: Add SoC level defaults for each type we support

In Fedora aarch64 we currently support A64/H5/H6 SoCs
so build the defaults for those so we can support one
SCP firmware per SoC rather than needing to build one
per device.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 configs/a64_defconfig | 2 ++
 configs/h5_defconfig  | 3 +++
 configs/h6_defconfig  | 2 ++
 3 files changed, 7 insertions(+)
 create mode 100644 configs/a64_defconfig
 create mode 100644 configs/h5_defconfig
 create mode 100644 configs/h6_defconfig

diff --git a/configs/a64_defconfig b/configs/a64_defconfig
new file mode 100644
index 0000000..d0ebe10
--- /dev/null
+++ b/configs/a64_defconfig
@@ -0,0 +1,2 @@
+CONFIG_PLATFORM_A64=y
+CONFIG_MFD_AXP20X=y
diff --git a/configs/h5_defconfig b/configs/h5_defconfig
new file mode 100644
index 0000000..1dfa31b
--- /dev/null
+++ b/configs/h5_defconfig
@@ -0,0 +1,3 @@
+CONFIG_PLATFORM_A64=y
+CONFIG_SOC_H5=y
+CONFIG_REGULATOR_GPIO_DRAM=y
diff --git a/configs/h6_defconfig b/configs/h6_defconfig
new file mode 100644
index 0000000..35b7f98
--- /dev/null
+++ b/configs/h6_defconfig
@@ -0,0 +1,2 @@
+CONFIG_PLATFORM_H6=y
+CONFIG_MFD_AXP20X=y
-- 
2.39.0

